#pragma once

#include "public.h"

#include <yt/core/rpc/public.h>

namespace NYT {
namespace NScheduler {

////////////////////////////////////////////////////////////////////////////////

NRpc::IServicePtr CreateJobTrackerService(TBootstrap* bootstrap);

////////////////////////////////////////////////////////////////////////////////

} // namespace NScheduler
} // namespace NYT

